knowledge_repo/app/static/css/pages/base.css

```diff
@@ -47,44 +68,73 @@
 }
 #navbar .navbar-form .input-group #searchbar {
     width: 100%;
-    background: #999;
-    color: #fff;
+    background: #ececec;
+    color: #000;
+    opacity: 1.0;
     border: none;
     transition: background 0.2s;
 }
 #navbar .navbar-form .input-group #searchbar:focus {
     background: #aaa;
     box-shadow: none;
     transition: background 0.3s;
+    background: #ececec;
 }
 #navbar .navbar-form .btn-default {
     border-color: whitesmoke;
+    background-color: #ececec;
+    border-color: #ececec;
 }
 #navbar .navbar-form .input-group #searchbar::-webkit-input-placeholder { /* Chrome */
-  color: #fff;
+  color: #000;
   opacity: 0.5;
-  font-style: italic;
+  /*font-style: italic;*/
 }
 #navbar .navbar-form .input-group #searchbar:-ms-input-placeholder { /* IE 10+ */
-  color: #fff;
+  color: #000;
   opacity: 0.5;
-  font-style: italic;
+  /*font-style: italic;*/
 }
 #navbar .navbar-form .input-group #searchbar::-moz-placeholder { /* Firefox 19+ */
-  color: #fff;
+  color: #000;
   opacity: 0.5;
-  font-style: italic;
+  /*font-style: italic;*/
 }
 #navbar .navbar-form .input-group #searchbar:-moz-placeholder { /* Firefox 4 - 18 */
-  color: #fff;
+  color: #000;
   opacity: 0.5;
-  font-style: italic;
+  /*font-style: italic;*/
 }
 .navbar a:hover {
-    background: #06c6b6;
-    opacity: 1;
-    border-color: #06c6b6;
+  background: #06c6b6;
+  opacity: 1;
+  border-color: #06c6b6;
+}
+
+
+.navbar-inverse .navbar-brand:focus, .navbar-inverse .navbar-brand:hover {
+    color: #000;
 }
+
+/* navbar click background color*/
+.navbar-inverse .navbar-nav>.open>a, .navbar-inverse .navbar-nav>.open>a:focus, .navbar-inverse .navbar-nav>.open>a:hover {
+  background-color: #ffffff;
```

Comment:
use `white` (i'll stop commenting this from here on out)

Reply:
👍 I'll update all of them